src/ej_users/routes.py

```diff
@@ -100,18 +108,89 @@ def logout(request):
     return HttpResponseServerError()
 
 
-@urlpatterns.route('profile/recover-password/')
-def recover_password(request):
+@urlpatterns.route('recover-password/<str:url_token>')
+def recover_password(request, url_token):
+
+    form = forms.RecoverPasswordForm.bind(request)
+    next = request.GET.get('next', '/login/')
+    isExpired = False
```

Comment:
Pra ficar um pouco mais seguro, seria legal se essass variavéis por default estivessem em True